demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

```diff
@@ -16,12 +15,3 @@
         <div>Courier: @Model.ShippingCourier</div>
     </div>
 </div>
-
-@*<div class="panel panel-default">
-    <div class="panel-heading">
-        <h3 class="panel-title">Shipment information</h3>
-    </div>
-    <div class="panel-body">
-
-    </div>
-</div>*@
```

Comment:
I think committing the .suo files is OK for a demo project, but the .sou for the ASP.NET MVC demo should not be changed as part of this PR. I'd prefer to have that change reverted in this branch (as well the change to this file).